corehq/apps/geospatial/models.py

```diff
@@ -58,9 +59,10 @@ class GeoConfig(models.Model):
 
     selected_disbursement_algorithm = models.CharField(
         choices=VALID_DISBURSEMENT_ALGORITHMS,
-        default=ROAD_NETWORK_ALGORITHM,
+        default=RADIAL_ALGORITHM,
         max_length=50
     )
+    _api_token = models.CharField(max_length=255, blank=True, null=True, db_column="api_token")
```

Comment:
Thoughts on using the [django_cryptography](https://django-cryptography.readthedocs.io/en/latest/examples.html) library? Seems like it could save us some code

Reply:
Just to avoid a conflict with the `api_token` property. I suppose I could've named it something else, but seeing as the token from the database (i.e. `_api_token`'s value) should never be used outside of the model class it made sense to name it as such.